DMTN-165.tex

```diff
@@ -50,7 +56,289 @@
+It is expected that users will still need to retrieve the full alert packet to be able to achieve their science goals.
+
+The size of the lightweight alert is $\sim$200 bytes/packet. 
+Averaged over 39 seconds per visit this implies an average bandwidth required of 400 kbps; to transmit all of the lightweight alerts in a visit within 5 seconds requires 3.2 Mbps.
+This implies that we can easily serve thousands of users the lightweight alert stream using a 10 Gbps network interface from the Data Facility.
+A night of observing would produce a total volume of lightweight alerts of 2\,GB.
+In contrast to the full alert stream, which is $\sim$ 820\,GB per night, it is easy to imagine a graduate student accessing the full alert stream from their laptop, or perhaps connecting a small or ad-hoc automated telescope to it.
+We believe that democratizing access to the alert stream will enable some of the most creative uses of the stream and ensure innovative use by scientists.
+
+Each lightweight alert packet would include a link to allow the user to retrieve the full alert packet containing all of the DPDD-specified quantities (image cutouts, \DIASource history, forced photometry where available, timeseries features, etc.).
+Rate limits assigned to each user would manage bandwidth use.
+The typical workflow will be for users to inspect the contents of the lightweight alert, filter them down to a relevant subset (e.g., alerts coincident with the LMC; alerts in deep drilling fields; fast-evolving transients; NEO-like alerts), and then request and retrieve the complete alert packets for that subset to further analyze.
+
+The discussion above focused on science users.
+Most (though perhaps not all) community alert brokers will still wish to retrieve all available alerts.
+In this design, brokers can simply be viewed as users with very high rate-limits, sufficient to allow transmitting all alerts out of the Data Facility within the 60-second latency window.
+%  Separate full stream?
+
+
+\begin{figure}
+  \begin{centering}
+  \includegraphics[width=\textwidth]{fig/lightweight_alert_dist.png}
+  \caption{Data flow diagram for the proposed lightweight alert system.
+  \label{fig:lightweight}}
+  \end{centering}
+\end{figure}
+
+\section{Technical Implementation}  
+
+% this is ... pretty sparse at the moment
+We consider a technical implementation derived from the current baseline as described in 
+\citeds{DMTN-093}.
+As the Alert Production pipeline runs, it would produce both the full-sized alert packets and their lightweight counterparts, both in Avro format.
+The lightweight alerts would be distributed to community brokers and science users using Kafka.
+The full-sized alerts would be stored on disk in an object store or other system accessible by HTTP.
```

Comment:
Although I can see how to make this work with HTTP, please consider using HTTPS.  Even though the Alert stream is world-public, we may need authentication to constrain bandwidth.  HTTPS can also ensure integrity (although I believe we will still want to sign lightweight and full-sized alerts).